### PR TITLE
Swd 10

### PR DESCRIPTION
**Pull request by nmcb02**

### CODE REVIEW

File: CUBE_IDE/VGA/Core/Inc/API_functions.h [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ *
+ * @brief  	Header file for API functions
+ *
+ * @authors Naomi Born
+ * @date    17-05-2023
+ * @version 1.0
+*********************************************************/
+#ifndef INC_API_FUNCTIONS_H_
+#define INC_API_FUNCTIONS_H_
+
```
Comment (Innocentkhajit) [inline CUBE_IDE/VGA/Core/Inc/API_functions.h:12]: no comments used? you did with the prototypes, but not the with the rest in the header file

File: CUBE_IDE/VGA/Core/Src/API_functions.c [hunk 2]
```diff
@@ -0,0 +1,109 @@
+/*******************************************************//**
+ * @file    API_functions.c
+ *
+ * @brief  	All API functions are created in this file
+ *
+ * @authors Naomi Born
+ * @date    17-05-2023
+ * @version 1.0
+*********************************************************/
+#include "API_functions.h"
```
Comment (Innocentkhajit) [inline CUBE_IDE/VGA/Core/Src/API_functions.c:10]: no comments refering what this is.

**Review by daniyeel — APPROVED** [review]
exquisite

**Review by Innocentkhajit — APPROVED** [review]
I am in awe.

**Pull request merged**